### ISSUE TITLE
Baseline Model

### ISSUE THREAD

**Issue by brandon-rutherford** [mattyli/capstone-federated-learning#8]

**Comment by DavidHoernke** [comment]
We have made a baseline model. A rudimentary first attempt has hit a DICE score of approximately 62% training on all 3000 images. We aim to improve this further.

Currently using a Unet architecture to identify the infection regions of covid infections

**Comment by DavidHoernke** [comment]
Some primary issues,
- Handling negative cases, this is similar to using DICE
- Data imbalance
- DICE score's validity as a evaluation metric
- Data augmentation
- Cross entropy and softmax torch fit together?
